renderer/stores/console.ts

```diff
@@ -0,0 +1,30 @@
+import { makeAutoObservable } from 'mobx';
+import { RootStore } from './root';
+
+export class ConsoleStore {
+  rootStore: typeof RootStore;
```

Comment:
There should be a better way to declare the type of `RootStore` but we'll just leave it like this for now.